pyproject.toml

```diff
@@ -49,6 +49,7 @@ select = [
   "UP031",
   "UP032",
   "UP034",
+  "S324", # avoid using insecure hash w/o acknowledgement
```

Comment:
I'd remove the comment because we don't have comments on other rules.

Reply:
makes sense